pkg/cli/idle/idle.go

```diff
@@ -693,7 +724,7 @@ func (o *IdleOptions) RunIdle() error {
 		return err
 	}
 
-	// annotate the endpoints objects to indicate which scalable resources need to be unidled on traffic
+	// annotate the service objects to indicate which scalable resources need to be unidled on traffic
```

Comment:
(might be worth splitting all of the "endpoints"->"service" comment fixes into a separate commit, since that's a belated fix to match how the code already was, rather than being associated with the new patch.)